docs/source/en/generation_strategies.md

```diff
@@ -21,7 +21,7 @@ more. It also plays a role in a variety of mixed-modality applications that have
 and vision-to-text. Some of the models that can generate text include
 GPT2, XLNet, OpenAI GPT, CTRL, TransformerXL, XLM, Bart, T5, GIT, Whisper.
 
-Check out a few examples that use [`~transformers.generation_utils.GenerationMixin.generate`] method to produce
+Check out a few examples that use [`~generation.GenerationMixin.generate`] method to produce
```

Comment:
The text was not being converted to a link. I've checked in the doc preview that it is now a link:
<img width="700" alt="Screenshot 2024-05-02 at 12 23 01" src="https://github.com/huggingface/transformers/assets/12240844/f8e5eed4-430e-465f-981e-e452141a4b28">